examples/nextjs-14-app-dir-si/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "nextjs-14-app-dir-rl",
```

Comment:
Wrong name. Change it with renaming the example. Don't forget to regenerate package-lcok